test/test_dynamic_shapes.py

```diff
@@ -3249,6 +3249,31 @@ def forward(self, arg0_1: "i64[2][1]cpu", arg1_1: "Sym(u2)", arg2_1: "Sym(u3)",
         self.assertEqual(result_compiled, result_eager)
         self.assertEqual(cnt.frame_count, 1)
 
+        # Pass a contiguous tensor. A recompilation will happen due to 0/1 speciialization on stride.
+        log_stream, ctx = logs_to_string(
+            "torch._functorch._aot_autograd.dispatch_and_compile_graph", "aot_graphs"
+        )
+        with ctx():
```

Comment:
is this logging for debugging or you meant to check it in?

Reply:
i meant to check it in .

Reply:
but you don't seem to use log_stream anywhere?